pom.xml

```diff
@@ -28,6 +30,7 @@
 
   <properties>
     <java.version>14</java.version>
+    <junitVersion>4.13.2</junitVersion>
```

Comment:
Be careful to remain coherent with other naming convention.
Here it seems we were using _dot notation_.

Reply:
Noted, thank you for the remarks 😊.